eZ/Bundle/EzPublishLegacySearchEngineBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,17 @@
+parameters:
+    ezpublish.search.legacy.connection.factory.class: eZ\Bundle\EzPublishLegacySearchEngineBundle\ApiLoader\ConnectionFactory
+    ezpublish.search.legacy.connection.class: Doctrine\DBAL\Driver\Connection
+
+services:
+    ezpublish.search.legacy.connection.factory:
+        class: %ezpublish.search.legacy.connection.factory.class%
+        arguments:
+            - @ezpublish.api.repository_configuration_provider
+        calls:
+            - [setContainer, [@service_container]]
+
+    ezpublish.api.search_engine.legacy.connection:
+        class: %ezpublish.search.legacy.connection.class%
+        factory_service: ezpublish.search.legacy.connection.factory
+        factory_method: getConnection
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/b7fbbef6477ef26e2d238e7afb5c336bdc33fc0f